packages/plugin-php/index.js

```diff
@@ -1,64 +1,57 @@
 import { PHP, PHP_FUNC } from '@octolinker/helper-grammar-regex-collection';
-import liveResolverQuery from '@octolinker/resolver-live-query';
 import resolverTrustedUrl from '@octolinker/resolver-trusted-url';
-import * as storage from '@octolinker/helper-settings';
 import mappingList from './mapping';
 
 export default {
   name: 'PHP',
 
   resolve(path, [target], meta, regExp) {
-    const enableBetterPHP = storage.get('enableBetterPHP');
-    if (enableBetterPHP) {
-      const linkMapping = mappingList.find(({ importPath }) =>
-        target.startsWith(`${importPath}\\`),
-      );
+    const linkMapping = mappingList.find(({ importPath }) =>
+      target.startsWith(`${importPath}\\`),
+    );
 
-      if (linkMapping) {
-        const { url, delimiter, hook } = linkMapping;
-        const targetPath = target.replace(/\\/g, delimiter);
+    if (linkMapping) {
+      const { url, delimiter, hook } = linkMapping;
+      const targetPath = target.replace(/\\/g, delimiter);
 
-        return resolverTrustedUrl({
-          target: url.replace('%s', targetPath),
-        });
-      }
+      return resolverTrustedUrl({
+        target: url.replace('%s', targetPath),
+      });
+    }
 
-      if (target.startsWith('Doctrine\\')) {
-        const targetPath = target.replace(/\\/g, '/');
-        const [, repo] = targetPath.split('/');
+    if (target.startsWith('Doctrine\\')) {
+      const targetPath = target.replace(/\\/g, '/');
+      const [, repo] = targetPath.split('/');
 
-        return resolverTrustedUrl({
-          target: `https://github.com/doctrine/${repo}/blob/HEAD/lib/${targetPath}.php`,
-        });
-      }
+      return resolverTrustedUrl({
+        target: `https://github.com/doctrine/${repo}/blob/HEAD/lib/${targetPath}.php`,
+      });
+    }
 
-      if (target.startsWith('PHPUnit\\')) {
-        const targetPath = target.replace(/\\/g, '/');
-        const [, ...filePath] = targetPath.split('/');
+    if (target.startsWith('PHPUnit\\')) {
+      const targetPath = target.replace(/\\/g, '/');
+      const [, ...filePath] = targetPath.split('/');
 
-        return resolverTrustedUrl({
-          target: `https://github.com/sebastianbergmann/phpunit/blob/master/src/${filePath.join(
-            '/',
-          )}.php`,
-        });
-      }
+      return resolverTrustedUrl({
+        target: `https://github.com/sebastianbergmann/phpunit/blob/master/src/${filePath.join(
+          '/',
+        )}.php`,
+      });
     }
 
     if (target.includes('\\')) {
       return [];
     }
 
     if (regExp === PHP_FUNC) {
-      return liveResolverQuery({
```

Comment:
For some reason, the ping through the OctoLinker API doesn't work for php.net anymore. A curl return 200, but a HEAD request made from nodejs fails with 503 `Service Temporarily Unavailable`. However, it should be fine to assume that the url we construct here are working in most cases like we do for the node.js core lib